src/filter/ztf.rs

```diff
@@ -383,9 +383,7 @@ pub async fn build_ztf_filter_pipeline(
     let (use_aliases_index, mut lsst_insert_aux_pipeline, lsst_aux_add_fields) =
         build_lsst_aux_data(use_aliases_index, filter_pipeline);
 
-    let mut aux_add_fields = doc! {
-        "aux": mongodb::bson::Bson::Null,
```

Comment:
Why was this removed? This was done in place of using an `unset` pipeline so we don't carry duplicated documents throughout the pipeline?

Overall I'm not sure I understand the changes made in the filter worker here...